### DISCUSSION TITLE
OpenID Connect - error with multiple providers

### DISCUSSION THREAD

**Discussion by nemanjavx** [pimcore/pimcore#12374] [Q&A / Support] [answered]
There is an issue with OpenID Connect bundle, where you can't add multiple sso providers because of the following JS error:

![image](https://user-images.githubusercontent.com/45720476/172797405-1b521056-a30e-47ee-9d60-11eaa8e6d82f.png)
![image](https://user-images.githubusercontent.com/45720476/172797726-7cb0d708-8485-44d3-abbf-d35cd8d01d4a.png)

The buttons don't render on the login page

**Comment by fashxp** [comment, marked as answer]
thx for reporting. will be fixed with next release, in the meantime you can use `1.0.x-dev`.
